code/mob/living/silicon/robot.dm

```diff
@@ -77,6 +77,13 @@
 	var/oil = 0
 	var/custom = 0 //For custom borgs. Basically just prevents appearance changes. Obviously needs more work.
 
+	//For borg_death_alert() and send_borg_death_alert()
+	var/message = null
+	var/mailgroup = "medresearch"
+	var/net_id = null
+	var/frequency = 1149
+	var/datum/radio_frequency/radio_connection
```

Comment:
At a glance. I don't think that all these vars need to exist on mob/living/silicon/robot and can probably be declared in the proc that builds the message or just made into defines.

Reply:
Ok I moved them into the proc, I also condensed borg_death_alert() and send_borg_death_alert() into one since it didn't really need to be two.

Reply:
ty, much better.